src/actions.js

```diff
@@ -101,20 +101,33 @@ export const types = {
   SHARED_ADDRESS_UPDATE: 'SHARED_ADDRESS_UPDATE',
   EXCEPTION_CAPTURED: 'EXCEPTION_CAPTURED',
   SET_FEATURE_TOGGLES: 'SET_FEATURE_TOGGLES',
+  // Feature Toggle actions
   FEATURE_TOGGLE_INITIALIZED: 'FEATURE_TOGGLE_INITIALIZED',
+  FEATURE_TOGGLE_UPDATE: 'FEATURE_TOGGLE_UPDATE',
+  FEATURE_TOGGLE_UPDATED: 'FEATURE_TOGGLE_UPDATED',
+  FEATURE_TOGGLE_READY: 'FEATURE_TOGGLE_READY',
+  FEATURE_TOGGLE_ERROR: 'FEATURE_TOGGLE_ERROR',
   SET_WALLET_CONNECT: 'SET_WALLET_CONNECT',
   SET_WALLET_CONNECT_MODAL: 'SET_WALLET_CONNECT_MODAL',
   SET_WALLET_CONNECT_SESSIONS: 'SET_WALLET_CONNECT_SESSIONS',
   SET_UNLEASH_CLIENT: 'SET_UNLEASH_CLIENT',
   WC_URI_INPUTTED: 'WC_URI_INPUTTED',
   WC_CANCEL_SESSION: 'WC_CANCEL_SESSION',
   WC_SET_CONNECTION_FAILED: 'WC_SET_CONNECTION_FAILED',
+  // Network Settings actions
+  NETWORKSETTINGS_UPDATE: 'NETWORK_SETTINGS_UPDATE',
+  NETWORKSETTINGS_UPDATE_SUCCESS: 'NETWORK_SETTINGS_UPDATE_SUCCESS',
+  NETWORKSETTINGS_UPDATE_FAILURE: 'NETWORK_SETTINGS_UPDATE_FAILURE',
```

Comment:
Actually I used a pattern here to frame the the names of actions in a taxonomy. See this commit message 6980cac80982b4437f57fc654920f27c38aa38eb. I can replicate it here. Basically we use a triple to communicate the `<domain>` or `<subject>`, then we inform the performed `<action>` in the domain, and we finish with the expected <result> of the action, being it an optional token in the action taxonomy.

```txt
I used the following taxonomy for the actions:
- <subject>_<action>[_<result>]

The `<subject>` and the `<result>` must be nouns and `<action>`
must be a verb.

For instance:

- NETWORKSETTINGS_UPDATE
- NETWORKSETTINGS_UPDATE_SUCCESS
- NETWORKSETTINGS_UPDATE_FAILURE
```

If we agree with this taxonomy, I will later refactor all the actions to match this rule. Otherwise I can follow your suggestion and change this to the current pattern.

I did it to try to bring more order to the actions names.

Reply:
I like it, but we should refactor the other ones

Maybe create an issue and also add a comment there referencing it, so we refactor it on the next time we change this file

Reply:
Fair. I will do it.

Reply:
The issue: https://github.com/HathorNetwork/hathor-wallet-mobile/issues/334